c2rust-transpile/tests/snapshots/snapshots__transpile@empty_init.c.snap

```diff
@@ -19,7 +19,7 @@ pub struct Scope {
 #[no_mangle]
 pub static mut scope: *mut Scope = &{
     let mut init = Scope {
-        next: 0 as *const Scope as *mut Scope,
+        next: ::core::ptr::null::<Scope>() as *mut Scope,
```

Comment:
We probably want to fix how we're generating this so that we just use `null_mut` directly, but this is fine for now, and still technically correct.